server/src/io/anuke/mindustry/server/ServerControl.java

```diff
@@ -211,19 +211,25 @@ private void registerCommands(){
             info("Stopped server.");
         });
 
-        handler.register("host", "<mapname> [mode]", "Open the server with a specific map.", arg -> {
+        handler.register("host", "[mapname] [mode]", "Open the server with a random or specific map.", arg -> {
```

Comment:
This is not a very good description for this command. It should be:
`Open the server. Will default to survival and a random map if not specified.`